### DISCUSSION TITLE
Firestick Shortcuts for TiVos

### DISCUSSION THREAD

**Discussion by tom76017** [GerryDazoo/Slinger#307] [Show and tell]
### Discussed in https://github.com/GerryDazoo/Slinger/discussions/111

<div type='discussions-op-text'>

<sup>Originally posted by **donscreen** October  2, 2022</sup>
I have two Tivo Premieres hooked to a Sllngbox 500 and a Tivo Premiere XL hooked to a Slingbox M2.  Gerry's server works perfectly.  I wanted to make URL shortcuts on a Firestick.  I use the android MX Player on the Firestick but it would not save the URL address as a playlist shortcut.  So I used VLC on my macbook to hook to each Tivo box and created a playlist (.m3u file extension) for each Tivo.  I then used ES File Explorer on the Firestick.  l ran it's FTP  server function to upload the saved playlist files Tivo1.m3u, Tivo2.m3u and Tivo3.m3u to the internal memory of the Firestick in a folder I just called Tivo.
The MX Player now has easy shortcuts to each of the 3 Tivo boxes via the 2 slingboxes.  We just then use browser shortcuts on a tablet or laptop for each separate remote control.   The Tivo Premiere XL remote uses code 68 for the Tivo button instead of 62 for the regular Tivo Premiere remote.
![IMG_8482](https://user-images.githubusercontent.com/114305703/193462251-a3c17153-3367-4c9e-ad66-37a802935ad9.JPG)</div>

**Comment by tom76017** [comment]
Worked for me.  I just moved this discussion to Show and Tell since it was lost in the "General" discussions.  VLC had audio sync issues.  MX Player was perfect in the Firestick 4K.

**Reply by RocketAlf** [reply]
Thanks for the information. A good solution (also) for the Echo Show 15 :-). Unfortunately it does not work for the older first generation Echo Show :-/. I guess I have to stay with my original solution for the older Echo Show ( integrate slinger as "camera" via RTSP server and monocle gateway) (?).  (--> https://github.com/GerryDazoo/Slinger/discussions/505#discussion-4783025)
